.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+Include a brief summary of the bug/feature/etc. that this PR seeks to address
+
+## Example / Current workflow
+Include a sample workflow to either **(a)** reproduce the bug with current codebase or **(b)** showcase the deficiency does this PR seeks to address
+
+## Bugfix / Desired workflow
+Include either **(a)** the same workflow from above with the correct output produced via this PR **(b)** some (pseudo)code containing the new workflow that this PR will (seek to) implement
+
+## Questions
+If there are open questions about implementation strategy or scope of the PR, include them here
+
+## Relevant issues
+If appropriate, please tag them here and include a quick summary
+
+## checklist
```

Comment:
Capitalize Checklist for consistency with other headings? (also for other two pull request templates)